-- Fix license field format for compatibility with current setuptools
-- The SPDX identifier format is not supported, need to use {text = ...} format

--- pyproject.toml.orig	2026-08-01 19:06:57 UTC
+++ pyproject.toml
@@ -12,7 +12,7 @@ classifiers = [
     "Programming Language :: Python :: 3",
     "Operating System :: OS Independent",
 ]
-license = "BSD-3-Clause"
+license = {text = "BSD-3-Clause"}
 dependencies = ["pybind11>3.0.0"]
 #dynamic = ["version"]
 version = "2026.05.0"
